test/parallel/test-fs-constants-windows.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+const common = require('../common');
+const fs = require('fs');
+const assert = require('assert');
+
+// Check if the two constants accepted by chmod() on Windows are defined.
+if (common.isWindows) {
+  assert.ok(fs.constants.S_IRUSR !== undefined);
+  assert.ok(fs.constants.S_IWUSR !== undefined);
```

Comment:
It'd be better to use `assert.notStrictEqual()` here. The "is windows?" guard is better written as:
```js
if (!common.isWindows)
  common.skip('Windows-only test');
```
Although in its current incarnation it's not really Windows-specific. You could just remove the guard altogether and rename the file.